signac/contrib/project.py

```diff
@@ -2387,14 +2387,20 @@ def get_project(cls, root=None, search=True, **kwargs):
         """
         if root is None:
             root = os.getcwd()
+
+        if not os.path.exists(root):
+            raise LookupError(
+                f"Unable to determine project id for nonexistent path '{os.path.abspath(root)}'."
+            )
+
         config = load_config(root=root, local=False)
         if "project" not in config or (
             not search
             and os.path.realpath(config["project_dir"]) != os.path.realpath(root)
         ):
             raise LookupError(
                 "Unable to determine project id for path '{}'.".format(
-                    os.path.abspath(root)
+                    f"Unable to determine project id for path '{os.path.abspath(root)}'."
                 )
```

Comment:
This is performing a string formatting twice -- you can use `"Error: {}".format(message)` or an f-string like `f"Error: {message}"` but not both.